pkg/scheduler/scheduler_test.go

```diff
@@ -201,6 +202,23 @@ func TestSchedulerCreation(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Failed to create scheduler: %v", err)
 	}
+
+	//Test case for：when registry merge one out of tree plugin which name already exists, create scheduler should fail.
```

Comment:
`When using an out of tree registry, if a plugin name has the same name as a default plugin, scheduler creation should fail.`

I think creating a registry that has that characteristic, rather than using `NewDefaultRegistry`, is a better test case.

Reply:
> `When using an out of tree registry, if a plugin name has the same name as a default plugin, scheduler creation should fail.`
> 
> I think creating a registry that has that characteristic, rather than using `NewDefaultRegistry`, is a better test case.

ok, i replace NewDefaultRegistry with a fake registry.

Reply:
please update the comment as suggested.

Reply:
> please update the comment as suggested.

ok, update comment.